CMakeLists.txt

```diff
@@ -119,6 +119,11 @@ whisper_option_depr(WARNING     WHISPER_SYCL                GGML_SYCL)
 whisper_option_depr(WARNING     WHISPER_SYCL_F16            GGML_SYCL_F16)
 whisper_option_depr(WARNING     WHISPER_CCACHE              GGML_CCACHE)
 
+if (GGML_CUDA)
```

Comment:
I think we also need to exclude MSVC from this as it will otherwise fail with:
```console
 error D8021: invalid numeric argument '/Wno-deprecated-gpu-targets'
 ```
 Some something like this should work:
 ```
 if (GGML_CUDA AND NOT MSVC)
    #GGML_CUDA enabled, add the necessary compile options -Wno-deprecated-gpu-targets
    add_compile_options(-Wno-deprecated-gpu-targets)
endif()
 ```